static/js/messages.js

```diff
@@ -0,0 +1,50 @@
+
+Vue.component('message-panel', {
+  template:
+  `<div class ="column">
+    <div class="tile is-parent">
+      <div class="tile box is-parent">
+        <div class="tile notification" v-bind:class="{'is-danger':fail, 'is-white':!fail}">
+          <div class="container">
+          <p>Message: </p> <input class="textbox" v-model="message"></input> Characters Left: {{45 - message.length}}
```

Comment:
The length can go negative and is not validated. Also, the input box is not wide enough.